enterprise/server/test/integration/remote_execution/rbetest/rbetest.go

```diff
@@ -366,12 +411,10 @@ func (r *Env) AddBuildBuddyServers(n int) {
 func (r *Env) AddBuildBuddyServerWithOptions(opts *BuildBuddyServerOptions) *BuildBuddyServer {
 	envOpts := &enterprise_testenv.Options{RedisTarget: r.redisTarget}
 	env := &buildBuddyServerEnv{TestEnv: enterprise_testenv.GetCustomTestEnv(r.t, envOpts), rbeEnv: r}
+	// We're using an in-memory SQLite database so we need to make sure all servers share the same handle.
+	env.SetDBHandle(r.testEnv.GetDBHandle())
```

Comment:
Made this change because I set up the userdb changes on the `rbetest.Env` instance, and realized that the DB instance there was different than the one that the app servers were using. Figured it was error prone for the `rbetest.Env` to have a different DB instance, so I updated things so that all the app servers inherit their DB from `rbetest.Env` instead of the first app server that gets created.